backend/dataall/modules/dataset_sharing/services/share_managers/s3_access_point_share_manager.py

```diff
@@ -431,54 +408,65 @@ def delete_target_role_access_policy(
         logger.info(
             'Deleting target role IAM policy...'
         )
-        access_point_name = S3AccessPointShareManager.build_access_point_name(share)
-        existing_policy = IAM.get_role_policy(
+
+        accesspoint_policy_name = ConsumptionRole.generate_policy_name(target_environment.environmentUri,
```

Comment:
can we name this the same as above to avoid confusion `share_resource_policy_name`

Reply:
changed